src/rabbit_delayed_message.erl

```diff
@@ -260,3 +267,17 @@ recover_exchange_and_bindings(#exchange{name = XName} = X) ->
                               "recovered bindings for ~s",
                              [rabbit_misc:rs(XName)])
     end).
+
+bump_routed_stats(ExName, Qs) ->
+    rabbit_global_counters:messages_routed(amqp091, length(Qs)),
+    [begin
+         QName = amqqueue:get_name(Q),
+         Key = {QName, ExName},
+         rabbit_core_metrics:channel_stats(queue_exchange_stats, publish, {self(), Key}, 1),
```

Comment:
would it cause any problem if the key contains the Pid of the delayed exchange process and not a channel process? Is this Pid used for something else than having a per-channel unique id?

Reply:
This is just an ID, metric aggregators do not interact with these pids.